_posts/0000-01-02-Robert-Gueydan.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Men's indignation, it seems, is more excited by legal wrong than by violent wrong.